machine/delta/delta_ag5648/busybox/patches/onie-sys-eeprom-platform.patch

```diff
@@ -0,0 +1,13 @@
+Add platform specific stuff for the sys-eeprom
+diff --git a/miscutils/sys_eeprom_i2c.c b/miscutils/sys_eeprom_i2c.c
+index ed3235b..46db9e3 100644
+--- a/miscutils/sys_eeprom_i2c.c
++++ b/miscutils/sys_eeprom_i2c.c
+@@ -55,6 +55,7 @@ int write_sys_eeprom(void *eeprom_data, int len)
+ 	    return -1;
+ 	}
+ 	eeprom_write_byte(&e, addr, *c);
++	usleep(5000);
```

Comment:
Patching busybox to change the write delay is no longer accepted.  Please use the busybox configuration parameter instead.  See commit 481b975ac4c5e8dbec255a2ffb1af3e383475d3a.